kolibri/core/logger/utils/user_data.py

```diff
@@ -540,17 +540,44 @@ def create_exams_for_classrooms(**options):
         )
         # everyone in the class has to take the exam
         for user in classroom.get_members():
-            # create exam log per user
-            examlog = ExamLog.objects.create(
-                exam=exam, user=user, completion_timestamp=now
+            random_seconds = random.randint(1, 500)
+            seconds = timezone.timedelta(seconds=random_seconds)
+            then = now - seconds
+            # create mastery log per user
+            sl = ContentSessionLog.objects.create(
```

Comment:
would help to distinguish from `summarylog`

```suggestion
            sessionlog = ContentSessionLog.objects.create(
```

Reply:
I did this so as not to rename a previous `sessionlog` above.

Reply:
I tell a lie - this was a hot fresh bowl of copy pasta.